package.json

```diff
@@ -9,14 +9,14 @@
     "Andrew Monks <a@monks.co>"
   ],
   "dependencies": {
-    "@blueoak/list": "^9.0.0",
-    "@npmcli/arborist": "^6.5.0",
+    "@blueoak/list": "^15.0.0",
```

Comment:
noting this has no minimum supported node version implications https://github.com/blueoakcouncil/blue-oak-list-npm-package